### DISCUSSION TITLE
Support DoH and Dot

### DISCUSSION THREAD

**Discussion by yoshino-s** [projectdiscovery/dnsx#741] [General]
### Please describe your feature request:
Support resolve dns by DoH or DoT, specific by `resolver ` options

### Describe the use case of this feature:
In some limited space, where dns packets are filtered, we can use doh or dot to resolve hosts

**Comment by Mzack9999** [comment]
@yoshino-s This is already supported via retryablehttp - Have a look at https://github.com/projectdiscovery/retryabledns/blob/main/doh/doh_client_test.go. It's enough to specify doh as protocol before the resolver.
